siteorigin-panels.php

```diff
@@ -925,7 +928,8 @@ function siteorigin_panels_render( $post_id = false, $enqueue_css = true, $panel
 		}
 	}
 
-	if( is_rtl() ) $panels_data = siteorigin_panels_make_rtl( $panels_data );
+	// A filter to change the panels_data right before the content is rendered.
+	$panels_data = apply_filters( 'siteorigin_panels_data_before_render', $panels_data, $post_id );
```

Comment:
That was the original intention, but the way we're reversing the cells now, on the processed version of the data, is a lot more effective.

https://github.com/siteorigin/siteorigin-panels/pull/133/files#diff-72e0505cdd6cb4ce7175f772c9c68bafR1018

It does mean this new filter isn't all that important though. We could remove it.